pom.xml

```diff
@@ -23,12 +23,13 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
         <flyway.version>4.2.0</flyway.version>
+        <logging.file>logs/spring-boot-logging.log</logging.file>
```

Comment:
let's name it ${project.name}.log, what do you think?

Reply:
My thoughts exactly